hack/demo/catalogd-metas-demo-script.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+#
+# Welcome to the catalogd metas API endpoint demo
+#
+trap 'trap - SIGTERM && kill -- -"$$"' SIGINT SIGTERM EXIT
+
+## until we cut a release containing the new API, we need to deploy local artifacts
```

Comment:
Why are some of these "double" commented?

Reply:
The linter said "use single quotes so they are not expanded now, but are expanded later", and the general syntax is "quote to capture strings with spaces / special characters".